base_layer/core/src/validation/header/header_full_validator.rs

```diff
@@ -41,6 +41,14 @@ use crate::{
 
 pub const LOG_TARGET: &str = "c::val::header_full_validator";
 
+// 999785e3bb6a43189c7236de7c9720df27d56f6b9724cd22f5115a097d3f770e is a bad block that bypassed validation due to a bug
+// in saving Monero seeds. The block uses a randomX  VM key of
+// 91ef83186cefaa646dc4c6e950e68e4debab52b4f4a9b7f465891e91fe5f6ce4, this key was used between height 2729 and 843 which
+// is about what you would expect from the Monero consensus. This block reuses this key at height 26320, which is a
+// violation of the Monero consensus rules. But in order to keep the network on the same chain, we whitelist this block
+// to bypass validation as that is the only validation it has failed.
```

Comment:
```suggestion
// to bypass validation as that is the only validation it has failed.
// Remove this block if Nextnet is reset
```

Reply:
Maybe we need to pin it to the specific genesis block hash in play, then it is not important if we omit to remove it.

Reply:
The comment should also talk about _header_ not _block_.